### PR TITLE
theflixer.tv

### PR DESCRIPTION
**Pull request by gork7777**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address
https://github.com/AdguardTeam/AdguardFilters/issues/131226

### Add your comment and screenshots

1. Your comment

Also there's share title.

2. Screenshots

<details>

<summary>Screenshot 1:</summary>

screen
</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Review by jellizaveta — APPROVED** [review]

**Comment by jellizaveta** [comment]
@gork7777 Where does this request appear?
`||evpdcnksmf.com^`

File: BaseFilter/sections/adservers.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@
 !
 !
 !
+||evpdcnksmf.com^
```
Comment (jellizaveta) [inline BaseFilter/sections/adservers.txt:12]: Where does this request appear?

**Review by jellizaveta — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/adservers.txt (hunk 1 above)
Comment (gork7777) [reply BaseFilter/sections/adservers.txt:12]: @jellizaveta I can't reproduce now, maybe blocking `||groodsuy.com^` is more useful?

Comment (Alex-302) [reply BaseFilter/sections/adservers.txt:12]: which script makes a request to this domain?

**Pull request closed without merging**